test/NuGet.Server.Tests/NuGet.Server.Tests.csproj

```diff
@@ -10,6 +11,8 @@
     <ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
Yes, the tooling adds it so I'd rather leave it than fight the tooling every time. Seem okay to you?